src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingSliderMessages/RequiredStakeMessage.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import Decimal from 'decimal.js';
+
+import Numeral from '~shared/Numeral';
+import { SLIDER_AMOUNT_KEY } from '../StakingInput';
+
+import styles from './RequiredStakeMessage.css';
+
+const displayName =
+  'common.ActionDetailsPage.DefaultMotion.StakingWidget.RequiredStakeMessage';
+
+const MSG = defineMessages({
+  requiredStake: {
+    id: `${displayName}.requiredStake`,
+    defaultMessage: ` ({stakePercentage}% of required)`,
+  },
+});
+
+const RequiredStakeMessage = () => {
+  const { watch } = useFormContext();
+  const sliderAmount = watch(SLIDER_AMOUNT_KEY);
+  const isUnderThreshold = sliderAmount < 10;
+  const isOverThreshold = !isUnderThreshold;
+  const nativeTokenDecimals = 18;
```

Comment:
This isn't necessarily true.

Granted, it is for 90% of the colonies, but the rest can have different amount of decimal lengths, and if the colony brought it's own token along, all bets are off, since that might not even have a decimal value, as it's not in the ERC20 spec _(hence why we have the `tokenDecimalsWithFallback` method, and why I hate having to support BYOT contract functionality)_